tests/fast-integration/kyma-environment-broker/client.js

```diff
@@ -0,0 +1,163 @@
+const axios = require("axios");
+const { 
+    debug,
+    getEnvOrThrow,
+} = require("../utils");
+
+const KYMA_SERVICE_ID = "47c9dcbf-ff30-448e-ab36-d3bad66ba281";
+
+class KEBConfig {
+  static fromEnv() {
+    return new KEBConfig(
+      getEnvOrThrow("KEB_HOST"),
+      getEnvOrThrow("KEB_CLIENT_ID"),
+      getEnvOrThrow("KEB_CLIENT_SECRET"),
+      getEnvOrThrow("KEB_GLOBALACCOUNT_ID"),
+      getEnvOrThrow("KEB_SUBACCOUNT_ID"),
+      getEnvOrThrow("KEB_PLAN_ID"),
+    );
+  }
+
+  constructor(host, clientID, clientSecret, globalAccountID, subaccountID, planID) {
+    this.host = host;
+    this.clientID = clientID;
+    this.clientSecret = clientSecret;
+    this.globalAccountID = globalAccountID;
+    this.subaccountID = subaccountID;
+    this.planID = planID;
+  }
+}
+  
+
+class KEBClient {
+  constructor(config) {
+    this.host = config.host;
+    this.clientID = config.clientID;
+    this.clientSecret = config.clientSecret;
+    this.globalAccountID = config.globalAccountID;
+    this.subaccountID = config.subaccountID;
+    this.planID = config.planID
+    this.serviceID = KYMA_SERVICE_ID;
+    
+    this._token = void 0;
```

Comment:
I don't like `void 0`. What is the purpose of this explicit assignment? For me just an unnecessary line that makes it harder to understand (I had to read 3 articles about why people used this construction in the previous version of ECMA script).

Reply:
Replaced with undefined. I like to have all properties declared in the class constructor just for transparency.